### PR TITLE
update/fix/last edit ru docs

### PR DESCRIPTION
**Pull request by h9lk**

### CODE REVIEW

File: docs/customization/modules.ru.md [hunk 1]
```diff
@@ -56,7 +63,10 @@ textures=high
 
 Обеспечивает совместимость со SourceTV.
 
-Настройка по умолчанию: **`sourcetv=on`** (все пресеты).
+* **Использование ГП:** отсутствует
+* **Использование ГП:** отсутствует
+
+Настройка по умолчанию: **`sourcetv=on`** (все песеты, *включая None*).
```
Comment (ghost) [inline docs/customization/modules.ru.md]: __[remark-lint]__ Emphasis should use `_` as a marker ([view](https://sider.review/gh/repos/69422496/pulls/601#issue-6154414))

<details>

| Rule | Severity |
| ---- | -------- |
| `emphasis-marker` | `warn` |

You can [close this issue](https://sider.review/gh/repos/69422496/pulls/601#issue-6154414) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

File: docs/customization/modules.ru.md [hunk 2]
```diff
@@ -65,16 +75,22 @@ textures=high
...
 ### Буфер снапшотов
 
-Управляет тем, насколько надежными будут считаться снимки, полученные от сервера, для применения буферизации / сглаживания на стороне клиента между полученными снимками. Также известна как линейная интерполяция (lerp или interp).
+Управляет тем, насколько надежными будут считаться снимки, полученные от сервера, для применения буферизации/сглаживания на стороне клиента между полученными снимками. Также известна как линейная интерполяция (lerp или interp).
+
+* **Использование ГП:** низкое
+* **Использование ГП:** отсутствует
 
-Настройка по умолчанию: **`snapshot_buffer=low`** (все пресеты).
+Настройка по умолчанию: **`snapshot_buffer=low`** (все пресеты, *включая None*).
```
Comment (ghost) [inline docs/customization/modules.ru.md]: __[remark-lint]__ Emphasis should use `_` as a marker ([view](https://sider.review/gh/repos/69422496/pulls/601#issue-6154416))

<details>

| Rule | Severity |
| ---- | -------- |
| `emphasis-marker` | `warn` |

You can [close this issue](https://sider.review/gh/repos/69422496/pulls/601#issue-6154416) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).

</details>

**Comment by Tiagoquix** [comment]
Please update your pull request to accommodate the latest changes. For reference: the `Sprays` section in `modules.md` was updated, and also in `modules.md` there was added a new section called `Contracts`.

Also, please update the `see_also.ru.md` file to accommodate the latest changes. View the changes to the original file here: https://github.com/mastercomfig/mastercomfig/commits/develop/docs/customization/see_also.md (from September 10 to this day).

Sorry for making you waste your time changing `*` to `_` in `modules.md`. I also changed them in the original file.

**Comment by h9lk** [comment]
> Please update your pull request to accommodate the latest changes. For reference: the `Sprays` section in `modules.md` was updated, and also in `modules.md` there was added a new section called `Contracts`.
> 
> Also, please update the `see_also.ru.md` file to accommodate the latest changes. View the changes to the original file here: https://github.com/mastercomfig/mastercomfig/commits/develop/docs/customization/see_also.md (from September 10 to this day).
> 
> Sorry for making you waste your time changing `*` to `_` in `modules.md`. I also changed them in the original file.

Hello, thanks for pointing out changes, actively working on this and translated almost everything with regards to edits - it's okay, have a linter, corrected it in 0.5 seconds

**Pull request closed without merging**